src/server.py

```diff
@@ -377,13 +352,12 @@ def run_jobtypes_query():
 def run_seta_query():
     start_date, end_date = clean_date_params(request.args, delta=SETA_WINDOW)
 
-    db = create_db_connnection()
-    cursor = db.cursor()
-    query = "select bugid, platform, buildtype, testtype, duration from testjobs \
-             where failure_classification=2 and date>='%s' and date<='%s'" % (start_date, end_date)
-    cursor.execute(query)
+    data = session.query(Testjobs.bugid, Testjobs.platform, Testjobs.buildtype, Testjobs.testtype,
+                         Testjobs.duration).filter(and_(Testjobs.failure_classification == 2,
+                                                        Testjobs.date >= start_date,
+                                                        Testjobs.date <= end_date)).all()
```

Comment:
@jmaher yeah, I don'y know why we can get nothing by this BETWEEN. But I try this on master branch and the pure sql doesn't works well also.  I guess it's because it not only compare the daytime but also compare the detail hours and seconds.(e.g 2016-7-24 1:12:2 is bigger than 2016-7-24 so can't been filter in by "<=2016-7-24") Maybe we could enlarge the data range? (e.g we want data between 2016-7-22 and 2016-7-24, so the data range should be 2016-7-21 to 2016-7-25)